plugins/python-build/bin/python-build

```diff
@@ -1516,6 +1517,60 @@ use_xcode_sdk_zlib() {
   fi
 }
 
+use_homebrew_tcltk() {
+  # get the version from the folder that homebrew versions
+  local tcltk_version_long="$(ls "$(brew --cellar tcl-tk)")"
```

Comment:
This looks like it would break horribly if Homebrew isn't present (e.g. in Linux).

Reply:
Actually, such a situation could use a test.

Reply:
This function is only called by `use_tcltk` if `local tcltk_libdir="$(brew --prefix tcl-tk 2>/dev/null || true)"` is defined which entails that the `brew`-command is available on the machine. If someone tries to call that function anywhere else it would break dozens of tests.